Source/sync.h

```diff
@@ -11,7 +11,7 @@
 namespace devilution {
 
 size_t sync_all_monsters(std::byte *pbBuf, size_t dwMaxLen);
-uint32_t OnSyncData(const TCmd *pCmd, const Player &player);
+size_t OnSyncData(const TSyncHeader &header, size_t maxCmdSize, const Player &player);

```

Comment:
Missing header includes

Reply:
I guess it's easy to miss because `sync.h` comes late in the alphabet. Since you already approved the PR, I think I might just merge what I have and fix this in a follow-up PR.